resolwe_bio/processes/workflows/chip_seq.yml

```diff
@@ -357,3 +357,341 @@
           tss: '{{input.rose_settings.tss}}'
           stitch: '{{input.rose_settings.stitch}}'
           mask: '{{input.rose_settings.mask}}'
+      - id: multiqc
+        run: multiqc
+        input:
+          data: '{{[input.case, steps.macs2]}}'
+
+- slug: workflow-macs2
+  name: "MACS2"
+  data_name: "{{ case|sample_name|default('?') }}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:chipseq:macs2rose2
+  category: Pipeline
+  input:
+    - name: case
+      label: Case (treatment)
+      type: data:alignment:bam
+    - name: control
+      label: Control (background)
+      type: data:alignment:bam
+      required: false
+    - name: promoter
+      label: Promoter regions BED file
+      type: data:bed
+      required: false
+      description: |
+        BED file containing promoter regions (TSS+-1000bp for example). Needed to get the number
+        of peaks and reads mapped to promoter regions.
+    - name: tagalign
+      label: Use tagAlign files
+      type: basic:boolean
+      default: false
+      description: |
+          Use filtered tagAlign files as case (treatment) and control
+          (background) samples. If extsize parameter is not set, run MACS
+          using input's estimated fragment length.
+    - name: prepeakqc_settings
+      label: Pre-peak QC settings
+      group:
+        - name: q_threshold
+          label: Quality filtering threshold
+          type: basic:integer
+          default: 30
+        - name: n_sub
+          label: Number of reads to subsample
+          type: basic:integer
+          default: 15000000
+        - name: tn5
+          label: TN5 shifting
```

Comment:
Make N small perhaps, something along the lines of `Tn5` (also used in description and on wikipedia)?